rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -69,11 +69,15 @@
 #endif
 
 // The stack space occupied is mainly dependent on the underling C standard library
+#ifdef OS_MAINSTKSIZE
+#define WORDS_STACK_SIZE OS_MAINSTKSIZE
+#else
```

Comment:
Yeah, there is 16K allocated for Ethernet in LWIP's case and 16K for USB. It would have been possible for us to use 16K reserved for USB for BufferedSerial and AT parser but that seemed to be an ugly solution with toolchain related ifdefs and might not have been portable. May be we need to rethink this. @kjbracey-arm